src/app/static/src/app/types.ts

```diff
@@ -220,3 +220,9 @@ export interface GenericChartMetadata {
 export interface GenericChartMetadataResponse {
     [key: string]: GenericChartMetadata;
 }
+
+export interface StepWarnings {
```

Comment:
Want to use this interface to type the getter that returns the warnings as well? Looks like it's untyped at the moment.